angular/demo/daisyui/src/app/samples/slider/slider.component.html

```diff
@@ -0,0 +1,14 @@
+<input
+	[auUse]="widget.directives.sliderDirective"
+	type="range"
+	[min]="state().min"
+	[max]="state().max"
+	[value]="state().sortedHandles[0].value"
+	[step]="state().stepSize"
+	[class]="'range'"
```

Comment:
Maybe
```angular
<input class="range" [class]="className"/>
```
Like this a user in stackblitz could use the slider like this:
```angular
   <app-slider min="0" max="100" stepSize="1" [values]="values" (valuesChange)="valuesChange($event)" className="range-success" />
```
and have a different color

Reply:
It is already the case, `className` is applied by `sliderDirective`